src/models/every-components-props.ts

```diff
@@ -0,0 +1,6 @@
+export default interface EveryComponentsProps {
```

Comment:
Move to component file

Reply:
I'm not sure what you want me to do in this situation. Should I move the interface in the App component or chop up the interface and put the properties of the interface inside the components that are using them?